### PR TITLE
Bybit editOrder (stopLossPrice, takeProfitPrice)

### PR DESCRIPTION
**Pull request by Dan-krm**
Added 'stopLossPrice' and 'takeProfitPrice' functionality to editOrder:
#13822

editContractOrder:
```
node examples/js/cli bybit editOrder '330160b7-8b71-4d3e-a4ae-ab5eb21e598f' BTC/USDT:USDT limit buy
0.002 20000 '{"takeProfitPrice":"24000","stopLossPrice":"19500"}'

bybit.editOrder (330160b7-8b71-4d3e-a4ae-ab5eb21e598f, BTC/USDT:USDT, limit, buy, 0.002, 20000, [object Object])
2022-06-16T04:33:43.575Z iteration 0 passed in 299 ms

{
  info: {
    ret_code: '0',
    ret_msg: 'OK',
    ext_code: '',
    ext_info: '',
    result: { order_id: '330160b7-8b71-4d3e-a4ae-ab5eb21e598f' },
    time_now: '1655354023.914515',
    rate_limit_status: '99',
    rate_limit_reset_ms: '1655354023911',
    rate_limit: '100'
  },
  id: '330160b7-8b71-4d3e-a4ae-ab5eb21e598f',
  order_id: '330160b7-8b71-4d3e-a4ae-ab5eb21e598f',
  stop_order_id: undefined
}
```

editUsdcOrder:
```
node examples/js/cli bybit editOrder '93570023-3b89-4268-8a5d-91c54acb8862' BTCPERP limit buy 0.011 
20000 '{"takeProfitPrice":"26000","stopLossPrice":"18900"}'

bybit.editOrder (93570023-3b89-4268-8a5d-91c54acb8862, BTCPERP, limit, buy, 0.011, 20000, [object Object])
2022-06-16T04:41:43.920Z iteration 0 passed in 283 ms

{
  info: {
    retCode: '0',
    retMsg: '',
    result: {
      orderId: '93570023-3b89-4268-8a5d-91c54acb8862',
      orderLinkId: '',
      createdAt: '1655354409083459',
      updatedAt: '0',
      symbol: 'BTCPERP',
      orderType: 'UNKNOWN',
      side: 'None',
      qty: '0.01100000',
      price: '20000.00',
      timeInForce: 'UNKNOWN',
      orderStatus: 'UNKNOWN',
      triggerPrice: '0.00'
    }
  },
  id: '93570023-3b89-4268-8a5d-91c54acb8862'
}
```

### CODE REVIEW

File: js/bybit.js [hunk 1]
```diff
@@ -3016,7 +3016,24 @@ module.exports = class bybit extends Exchange {
...
             request['orderPrice'] = this.priceToPrecision (symbol, price);
         }
-        const method = market['option'] ? 'privatePostOptionUsdcOpenApiPrivateV1ReplaceOrder' : 'privatePostPerpetualUsdcOpenApiPrivateV1ReplaceOrder';
+        if (market['swap']) {
+            const stopLossPrice = this.safeValue2 (params, 'stopLoss', 'stopLossPrice');
+            const isStopLossOrder = stopLossPrice !== undefined;
+            const takeProfitPrice = this.safeValue2 (params, 'takeProfit', 'takeProfitPrice');
+            const isTakeProfitOrder = takeProfitPrice !== undefined;
+            if (isStopLossOrder) {
+                request['stopLoss'] = stopLossPrice;
```
Comment (carlosmiei) [inline js/bybit.js:3025]: **priceToPrecision** missing for stopLoss/takeProfit

File: js/bybit.js [hunk 2]
```diff
@@ -3074,6 +3090,21 @@ module.exports = class bybit extends Exchange {
             // inverse swaps
             method = isConditionalOrder ? 'privatePostV2PrivateSpotOrderReplace' : 'privatePostV2PrivateOrderReplace';
         }
+        const stopLossPrice = this.safeValue2 (params, 'stop_loss', 'stopLossPrice');
+        const isStopLossOrder = stopLossPrice !== undefined;
+        const takeProfitPrice = this.safeValue2 (params, 'take_profit', 'takeProfitPrice');
+        const isTakeProfitOrder = takeProfitPrice !== undefined;
+        if (isStopLossOrder) {
+            request['stop_loss'] = stopLossPrice;
```
Comment (carlosmiei) [inline js/bybit.js:3098]: same here and below

**Comment by frosty00** [comment]
#13859 check how i've done it here

**Comment by samgermain** [comment]
@frosty00 Why was this closed? Doesn't it just need edits?

**Pull request closed without merging**